src/SymphonyOSS.RestApiClient/Generated/OpenApi/AgentApi/AgentApi.cs

```diff
@@ -4625,6 +4667,11 @@ public string BaseUrl
                     }
                 }
             }
+            catch (Exception e)
+            {
+                _log.LogError(0, e, "An error occured while trying to send the request to symphony.");
```

Comment:
I think you might have to go something like this here as well:

_log.LogError("Unable to launch monster data flusher, ex={0}", ex);